CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ------
 **ENHANCEMENTS**
 
+- Add validation to prevent `cluster_resource_bucket` creating not in same region as cluster.
```

Comment:
How about "Add validation to prevent using a `cluster_resource_bucket` that is in a different region than the cluster"

Reply:
Good idea, done!